tests/lib/rules/max-lines.js

```diff
@@ -233,6 +240,50 @@ ruleTester.run("max-lines", rule, {
                 }
             ]
         },
+        {
+
+            // Questionable. Makes sense to report this, and makes sense to not report this.
```

Comment:
Did this behavior exist before your change? I'd be in favor of assuming `max: 0` is a mistake and normalizing to `max: 1`. I can't think of a reason we'd ever want 0 max lines in a file.

Reply:
Yes, that's behavior in the actual version as well, I thought it would be better not to change that behavior in this PR and just added this test to make sure that this change doesn't affect `max: 0` on an empty file, since that isn't technically an extra empty line after a linebreak. 

I'm not sure why is it allowed to set `max: 0`, but it seems to be working like that from the start, and I couldn't find any concerns regarding that in the initial PR. There is also an existing [test with `max: 0`](https://github.com/eslint/eslint/blob/cc468c01021385a028de727eefcd442e7f34875c/tests/lib/rules/max-lines.js#L238).

The only use case for `max: 0` I could think of is using ESLint to disallow any `.js` files in some directories, and for that cause it might make sense to treat an empty file as a file with 1 line.

Reply:
Okay, I agree with not changing what it’s already doing. 👍